src/components/view-pane/index.tsx

```diff
@@ -1,29 +1,31 @@
 import * as React from 'react';
 import * as CSSModules from 'react-css-modules';
-import {connect} from 'react-redux';
-import {InlineData} from 'vega-lite/build/src/data';
-import {SortField, SortOrder} from 'vega-lite/build/src/sort';
-import {FacetedCompositeUnitSpec} from 'vega-lite/build/src/spec';
-import {ActionHandler, createDispatchHandler} from '../../actions/redux-action';
-import {ShelfAction} from '../../actions/shelf';
-import {SHELF_AUTO_ADD_COUNT_CHANGE, SHELF_GROUP_BY_CHANGE} from '../../actions/shelf/index';
-import {SPEC_FIELD_PROP_CHANGE} from '../../actions/shelf/spec';
-import {State} from '../../models';
-import {Bookmark} from '../../models/bookmark';
-import {VoyagerConfig} from '../../models/config';
-import {Result} from '../../models/result/index';
-import {ShelfFilter} from '../../models/shelf/filter';
-import {SHELF_GROUP_BYS, ShelfGroupBy} from '../../models/shelf/index';
-import {selectBookmark, selectConfig, selectMainSpec} from '../../selectors';
-import {selectFilteredData} from '../../selectors/index';
-import {selectResult} from '../../selectors/result';
+import { connect } from 'react-redux';
```

Comment:
Please correct your editor's formatter to not have space between braces so we have consistent import -- otherwise we will keep overriding each other all the time.  

Perhaps, add to project config so everyone can benefit from this :)